backend/lambda/database_calls.py

```diff
@@ -65,22 +75,37 @@ def grab_deepcite_entry(self, id):
         print("Deepcite entry responses:", responses)
         return responses
 
-    def record_call(self, new_submission, base_id, user_id, stage, status_code, response, time_elapsed, versions):
+    def record_call(self, existing_id, base_id, user_id, stage, status_code, response, time_elapsed, versions):
         try:
             with self.conn.connect() as cur:
-                if new_submission:
+                if existing_id==None: # This is a new entry
```

Comment:
[Minor] but I think in pep8 they recommend comparing to singletons with `is`.
https://www.python.org/dev/peps/pep-0008/#programming-recommendations